src/pages/picture-of-the-day/picture-of-the-day.jsx

```diff
@@ -0,0 +1,117 @@
+import { React } from "react";
+import { Box, Typography } from "@mui/material";
+import { Footer, Header, Loading, PictureDatePicker } from "../../components";
+import Moon from "./../../assets/moon.png";
+import styled from "styled-components";
+import { useGetData } from "../../hooks/useGetData";
+import Wizetronaut from "./../../assets/wizetronaut.png";
+import WizetronautError from "./../../assets/wizetronaut-error.png";
+
+const PageContainer = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0px 50px;
+`;
+
+const MainContainer = styled.div`
+  background-color: #cacaca70;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+`;
+
+const ImageContainer = styled.div`

```

Comment:
the styled components could live in their own file, maybe `picture-of-the-day.styled.jsx`, this can help to keep the file of the main component clean